optimade/server/query_params.py

```diff
@@ -214,7 +213,7 @@ def __init__(
             ge=0,
         ),
         page_number: int = Query(
-            0,
+            None,
             description="RECOMMENDED for use with _page-based_ pagination: using `page_number` and `page_limit` is RECOMMENDED.\nIt is RECOMMENDED that the first page has number 1, i.e., that `page_number` is 1-based.\nExample: Fetch page 2 of up to 50 structures per page: `/structures?page_number=2&page_limit=50`.",
             ge=0,
```

Comment:
```suggestion
            ge=1,
```

Given the description, I think this makes sense.